code/API_definitions/connected-network-type.yaml

```diff
@@ -0,0 +1,405 @@
+openapi: 3.0.3
+info:
+  title: Connected Network Type
+  description: |
+    This API provides the customer with the ability to query to which Mobile Communication Technology it is connected to.
+
+    # Introduction
+
+    ## Connected Network Type
+
+    API consumer is able to inquire device's connected network type.
+
+    # Relevant terms and definitions
+
+    * **Device**: A device refers to any physical entity that can connect to a network and participate in network communication.
+      At least one identifier for the device (user equipment) out of four options: IPv4 address, IPv6 address, Phone number, or Network Access Identifier assigned by the mobile network operator for the device.
+
+    * **Network Type** : Network Type is intended to provide insight to connected network's capabilities from standards perspective.  Actual network capabilities may differ based on implementation and MUST be checked with the connected network provider.
```

Comment:
```suggestion
    * **Network Type** : Network Type is intended to provide insight into the connected network's capabilities from standards perspective, and to reflect the radio technology that would be displayed by the device to the end user where applicable.  Actual network capabilities may differ based on implementation and MUST be checked with the connected network provider.
```

Reply:
@eric-murray, as you suggested in another change to this PR, identifying the connected technology based connected radio cell is only one option.  For some implementations it may not be possible to accurately identify the connected radio technology.  Would you be ok with the following change: ' ... from standards perspective, and to reflect the radio _**or core network**_ technology that would be displayed ...'

Reply:
@eric-murray  
I was thinking of the NSA cases where connected radio and core technology may not be the same.  In any case, generalizing is the way to go.